### ISSUE TITLE
Some code widget buttons break for empty fenced code

### ISSUE THREAD

**Issue by Maarrk** [silverbulletmd/silverbullet#749]
This is a result of fix for #734:

https://github.com/silverbulletmd/silverbullet/blob/cfd1e53f621b8ad435054fb755d7b4026a810bcb/plugs/query/widget.ts#L55-L57

Both "Edit" and "Bake" buttons break, but "Reload" or baking the whole page work.

Indeed not unlikely, except for Table of Contents added manually 😄 
~~~
```toc
```
~~~

Workaround: Write anything inside, which will be ignored by `toc` anyway.

Proposal: Make the manual created contain the text inside, like:
~~~
```toc
Table of Contents
```
~~~
for current behaviour, but it would also allow self-translation
~~~
```toc
Spis treści
```
~~~

To not worsen the UX for English speaker, the first snippet could be available as `/toc`

*PS: I feels guilty to derive features based on implementation details*

**Comment by zefhemel** [comment]
Right, this is an edge case I didn't think of as part of my "fix" won't affect a lot of people, but still a bit silly.
